draft-ietf-taps-impl.md

```diff
@@ -556,10 +556,19 @@ not using TCP), the action should fail gracefully. The application may be inform
 The Transport Services implementation should allow protocol instances in the Protocol Stack to pass up arbitrary generic or protocol-specific
 errors that can be delivered to the application as Soft Errors. These allow the application to be informed of ICMP errors, and other similar events.
 
+## Pooled Connection {#pooled-connections}
+
+For message reordering resistant request/response protocols like HTTP, interactions can be automatically distributed across several underlying transport connections. 
```

Comment:
```suggestion
For protocols that require in-order request/response pairs, like HTTP, interactions can be automatically distributed across several underlying transport connections. 
```

Reply:
Thank you for your suggestion – I think you got me wrong, as I wanted to stress that there is no ordering requirement across different requests, but the ordering between request/response  is certainly ordered.
I update the text to make this clearer.